src/tooltip.js

```diff
@@ -190,8 +190,14 @@ class HoverTooltip extends Tooltip {
         el.addEventListener("blur", function() {
             if (!el.contains(document.activeElement)) this.hide();
         }.bind(this));
-        
-        el.addEventListener("wheel", event.stopPropagation);
+
+        el.addEventListener("wheel", function(event) {
+            event.stopPropagation();
+            var contentOverflows = el.scrollHeight > el.clientHeight;
+            if (!contentOverflows) {
```

Comment:
can we use  event.target here, rename this function to something like preventParentScroll move to lib and use in both places?